app/src/main/java/com/mobiledevelopmentfoodapp/RetrofitClientIntance.kt

```diff
@@ -5,15 +5,14 @@ import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitClientInstance {
 
-    private var retrofit: Retrofit? = null;
-    private val BASE_URL = "https://pkgstore.datahub.io/core/country-list/data_json/data/8c458f2d15d9f2119654b29ede6e45b8/data_json.json/"
+    private var retrofit: Retrofit? = null
```

Comment:
Proper formatting can be attained using the hotkey ctr+alt+l on windows. Formatting best practices can be found at https://kotlinlang.org/docs/coding-conventions.html#formatting